<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArkMidnight</class>
 <widget class="QMainWindow" name="ArkMidnight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ArkMidnight</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>780</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>780</width>
     <height>540</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>780</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>540</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>780</width>
        <height>540</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>780</width>
        <height>540</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/new/prefix1/image/main.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_start">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_guide">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>GUIDE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>540</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>780</width>
        <height>540</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>780</width>
        <height>540</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/new/prefix1/image/main.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_return_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>19</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>631</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color:rgb(233,247,168);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color:rgb(233,247,168);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>181</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>29</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/stage/image/stages/stage_1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>181</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>29</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/stage/image/stages/stage_2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>181</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>29</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/stage/image/stages/stage_3.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_return">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>19</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
